Big update lots of stuff changed

LiquidWater opened the pull request:
More details in commit logs

LiquidWater committed:
Merge pull request #1 from JulianChow94/master

Update fork

LiquidWater committed:
Comments and imports minor fix, prepping for MatchDetailsService

LiquidWater committed:
Added models for MatchDetailsRequest and skeleton code for MatchDetailsService

LiquidWater committed:
Finished models for match details

Moved UnixTimeStampToDateTime function to a shared

LiquidWater committed:
fixed typo

LiquidWater committed:
Uses a CS file to hold strings instead of external json

JulianChow94 reviewed (changes requested):
Hey this looks good (assuming you tested it) but theres something I want to address here and maintain for future PRs. You've been putting alot of dev comments into the code and I'd like to avoid that.

I'm assuming we all have our own source for the API documentation so I really dont think its necessary to repeat that documentation in the code, nobody should be working on this project without that literature anyway. The internal logic of the code itself should be clear and concise enough to explain what its doing without any further clarification (We dont have alot of devs here so thats definitely a goal we should aim for).

With regards to commented out code, it may be useful in the future but its essentially dead code. We can keep local documents on them (and share them through some other means) but I'd like to keep dead/inactive code out of our solution.

Let's try to limit code comments to:
- TODOs
- Deletion Flags

LiquidWater committed:
Code cleanup

Removed some comments to comply with PR request, removed InitializeUrl function as its no longer needed.

LiquidWater committed:
Codemaid automated code cleanup

JulianChow94 commented:
LGTM, merging
